apps/condo/domains/common/utils/serverSchema/index.js

```diff
@@ -93,6 +93,13 @@ class GqlWithKnexLoadList {
         this.keystone = modelAdapter
         this.knex = modelAdapter.adapter.knex
     }
+    async loadAggregate (rawAggregate, ids) {
+        const knexQuery = this.knex(`${this.listKey}`)
```

Comment:
Is there a way not to be tied up on knex? I believe we still have some plans to move to `prisma` ...

Reply:
I don't think so. There are not many raw SQL querries in project and we don't have good solution for this

Reply:
then add todo to migrate to prisma